spec/spec_helper.rb

```diff
@@ -8,14 +8,13 @@
 require 'capybara/rails'
 require 'capybara-webkit'
 
-require "factory_girl_rails"
+require 'factory_girl_rails'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.